Classes/Private/Manager/OCTManager.m

```diff
@@ -27,6 +28,7 @@ @interface OCTManager () <OCTToxDelegate, OCTSubmanagerDataSource>
 @property (strong, nonatomic, readwrite) OCTSubmanagerChats *chats;
 @property (strong, nonatomic, readwrite) OCTSubmanagerAvatars *avatars;
 @property (strong, nonatomic, readwrite) OCTSubmanagerFiles *files;
+@property (strong, nonatomic, readwrite) OCTSubmanagerCalls *managerCalls;
```

Comment:
You are right about the `managerCalls` and `callsManager`, it should be consistent. For not naming it `calls`, I thought it would be confusing since we have OCTCalls and that OCTSubmanagerCalls has a public property `calls` as well. So I thought it would be beneficial to differentiate the two. 

What do you think? Should I just change it to be consistent with the others instead?